src/os/macosx/macos.mm

```diff
@@ -66,6 +72,10 @@ void GetMacOSVersion(int *return_major, int *return_minor, int *return_bugfix)
 	}
 
 #if (MAC_OS_X_VERSION_MIN_REQUIRED < MAC_OS_X_VERSION_10_10)
+#ifdef __clang__
+#	pragma clang diagnostic push
+#	pragma clang diagnostic ignored "-Wdeprecated-declarations"
+#endif
```

Comment:
Is it not more worthwhile to replace the deprecated calls with something not-deprecated, rather than just hiding the warnings?

Reply:
There is not alternative unless you strictly allow only OSX 10.10 or later.

Reply:
#ifdef ? :>